rllib/algorithms/dqn/dqn.py

```diff
@@ -246,6 +250,11 @@ def training(
                 prioritized_replay_eps: Epsilon parameter sets the baseline probability
                 for sampling so that when the temporal-difference error of a sample is
                 zero, there is still a chance of drawing the sample.
+            td_error_loss_fn: "huber" or "mse". loss function for calculating TD error
+                when num_atoms is 1. Note that if num_atoms is > 1, this parameter
+                is simply ignored, and softmax cross entropy loss will be used.
+            categorical_distribution_temperature: Set the temperature parameter used
```

Comment:
Make sure it's clear to the user that this is only during evaluation. during training we simply do argmax, right?

Reply:
done